backend/tools/get_slots_in_week.py

```diff
@@ -0,0 +1,14 @@
+from db.models import Slots, SlotsShema
+from datetime import timedelta
+from tools.transform_str_to_datetime_and_back import transform_string_to_datetime, transform_datetime_to_string
+
+
+def get_slots_in_week(date):
```

Comment:
this function is similar to get_slots_in_week_by_filter. Would it be possible to merge them somehow?